docker/runtime/Dockerfile.pytorch

```diff
@@ -17,7 +19,9 @@ RUN apt-get update -y \
 
 RUN python3 -m pip install pip --upgrade
 RUN python3 -m pip install onnx==1.11.0
-RUN python3 -m pip install /install/python/torch_blade*.whl -f https://download.pytorch.org/whl/torch_stable.html
+ADD ./pytorch_blade/scripts/pip/requirements-dev-${TORCH_VERSION}.txt /install/python/requirements.txt
+RUN python3 -m pip install -r /install/python/requirements.txt -f https://download.pytorch.org/whl/torch_stable.html
+RUN python3 -m pip install /install/python/torch_blade*.whl 
```

Comment:
Do you want to add `pytorch` wheel as requirements of `torch_blade` wheel? that we do not need to install a specify PyTorch version manually.

Reply:
It's ok to add torch as a dependence of torch_blade. However torch_blade doesn't depend on torchvision, in order to install the correct version of torchvision, we had to specify it in the requirements.txt